packages/components/src/components/wallet-icon/wallet-icon.tsx

```diff
@@ -70,6 +72,7 @@ const WalletIcon = ({ gradient_class, icon, size = 'medium', type, has_bg, hide_
             {(type === 'demo' || type === 'crypto') && (
                 <Icon icon={icon} width={sizes.box[size].width} height={sizes.box[size].height} />
             )}
+            {type === 'cfd' && <TradingPlatformIcon icon={icon} size={sizes.square[size]} />}
```

Comment:
@aizad-deriv Honestly I don't understand why we need to add `TradingPlatformIcon` here if we can use `Icon` component as before and just pass right icon name, for example 'IcMt5SwapFreePlatform'. You don't need to change this component and even don't need to create `TradingPlatformIcon` as for me